source/_integrations/hisense_aehw4a1.markdown

```diff
@@ -0,0 +1,43 @@
+---
+title: "Hisense AEH-W4A1"
+description: "Instructions to setup the Hisense AEH W4A1 wifi module for ACs."
+logo: hisense.png
+ha_release: 0.102
+ha_category:
+  - Climate
+ha_iot_class: Local Pull
+---
+
+The Hisense AEH-W4A1 is wifi module used to give wifi connectivity to some Hisense ACs and rebranded models (Smart Cool, Beko and others).
+
+Later AC models use other wifi modules (like AEH-W4B1 and AEH-W4E1) that are not compatible or otherwise tested  with this integration.
+
+It is highly recommended not to use one of the official smartphone applications and this integration at the same time, to avoid problems related to command collision.
+
+
+## Configuration
+
+To integrate one or more Hisense AEH-W4A1 controlled ACs with Home Assistant, use the Configuration/Integrations or add the following section to your `configuration.yaml` file:
+
+```yaml
+# Automatic discovery
+hisense_aehw4a1:
+```
+
+Alternately, devices that are not discoverable can be statically configured.
```

Comment:
Please move addition examples in a separate section below the `{% configuration %}` block.

Reply:
Only the example configuration or all the text?

Reply:
I would say the text that belongs to the example should go down with the example?